README.md

```diff
@@ -1,3 +1,43 @@
-# Membership
+# Node.js Foundation - Membership
 
-Repo to work out the system for individual membership in the foundation.
+## What types of members are there?
+
+* Enterprise
+* Individual
+* Student
+* Active Contributor
+
+## What are the conditions of eligibility?
+
+TBD - Join the discussion on this at [issues/8](https://github.com/nodejs/membership/issues/8)
+
+## How to join
+
+To apply to be a member of the Node Foundation, create a new issue in this repository. Use the formatting given below, and preview your post before you submit it to make sure formatting and content are correct.
+
+### Application structure:
```

Comment:
That's fair play - I thought the goal with application was to simply ensure members have an intent to develop, maintain, and, more generally, contribute to the Node community. Could be wrong on that, though. Let's keep this discussion in https://github.com/nodejs/membership/issues/6, to make sure all points are represented at a single location.

I have quoted your comment in https://github.com/nodejs/membership/issues/6 to maintain the integrity of the conversation.